examples/basics/src/lib.rs

```diff
@@ -0,0 +1,104 @@
+#[macro_use]
+extern crate neon;
+
+// thanks to @rm-rf-etc for these: https://github.com/neon-bindings/neon/issues/260
```

Comment:
(Yeah, this is such an awful foot-gun in Neon 0.1. We're fixing this in 0.2, which will be shipping very soon. In the meantime this is good, and we can update this example and get rid of the comment once it's no longer necessary. That change won't invalidate the rest of this example project, which will still be useful!)